Dockerfile

```diff
@@ -24,14 +24,6 @@ CMD start
 
 FROM deps as app
```

Comment:
It seems that this whole build stage can be removed - could you try? We should probably rename deps to app.

Reply:
Okay.